package/boot/uboot-layerscape/files/fsl_ls1028a-rdb-sdboot-uEnv.txt

```diff
@@ -0,0 +1,9 @@
+fdtaddr=0x8f000000
+loadaddr=0x81000000
+fdt_high=0xffffffffffffffff
```

Comment:
Please fix ```fdt_high``` value for proper work fit image.

Reply:
Ok. Thanks.

Reply:
Are You sure, that this board boot with `fdt_high=0xffffffffffffffff`?